code/modules/surgery/burn_dressing.dm

```diff
@@ -0,0 +1,175 @@
+
+/////BONE FIXING SURGERIES//////
+
+///// Repair Hairline Fracture (Severe)
+/datum/surgery/debride
+	name = "Debride and repair burnt flesh"
+	//steps = list(/datum/surgery_step/debride, /datum/surgery_step/disinfect, /datum/surgery_step/regenerate_flesh, /datum/surgery_step/dress)
+	steps = list(/datum/surgery_step/debride, /datum/surgery_step/dress)
+	target_mobtypes = list(/mob/living/carbon/human)
```

Comment:
guess xenos and monkeys are just shit outta luck if they get a bad burn wound, huh?

Reply:
same with the other surgeries you've added

Reply:
Good, xenos deserve to be easier to take down

Reply:
but what about lone/staff of change xenos?

what if the queen gets a bad burn on their leg?

you're just kinda... fucked

Reply:
"I don't like this type of antag" isn't an excuse to intentionally make a system not account for the existence of/accommodate said type of antag